flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -907,7 +887,7 @@ private ApplicationReport startAppMaster(
 				}
 
 				final String jobGraphFilename = "job.graph";
-				flinkConfiguration.setString(JOB_GRAPH_FILE_PATH, jobGraphFilename);
```

Comment:
This is `flinkConfiguration`, indeed. We pass `flinkConfiguration` as `configuration` in this scope and I think either we don't pass it and use `flinkConfiguration` as context value or we use `configuration`. But never mix up.